VS2017/Source/Rocks.cpp

```diff
@@ -0,0 +1,39 @@
+#include "Rocks.h"
+
+Rocks::Rocks(const Shader& shader, Object& cubeObject) : ComplexModel(shader)
+{
+	shader.Bind();
+
+
+	// 4 cube base with single cube on top
+	glm::vec3 translation(1.0f, 1.0f, 1.0f);
+	glm::vec3 rotation(1.0f, 1.0f, 1.0f);
+	float angle = 0.0f;
+	glm::vec3 scale(0.5f, 0.5f, 0.5f);
+	glm::vec4 limeGreen(0.196f, 0.804f, 0.196f, 1.0f);
+	glm::vec4 yellow(0.8f, 0.7f, 0.0f, 1.0f);
+	glm::vec4 dirtBrown(0.3f, 0.15f, 0.03f, 1.0f);
+	glm::vec4 dimGray(0.412f, 0.412f, 0.412f, 1.0f);
+
+	float initialX = 0;
+	float initialY = 0;
+	float initialZ = 0;
+
```

Comment:
what are these used for?

Reply:
> what are these used for?
those are probably used for the transformations of the matrices

Reply:
the X, Y and Z coordinates and any changes that we wish to apply to them

Reply:
My question was more, why is it necessary, it's hardcoded as 0

Reply:
Let's say we want to choose where to put these objects, we could have a way or methods of changing this value, possibly for randomization of the placement. I just hardcoded it as 0 for the time being so that its there and less work to add the modification to the position to each vector later on.